<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeakCanSetupPage</class>
 <widget class="QWidget" name="PeakCanSetupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Driver:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="laDriver">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Interface:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="laInterfaceName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>Bitrate:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbBitrate"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="laOptions">
     <property name="text">
      <string>Options:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbListenOnly">
       <property name="text">
        <string>Listen only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAutoRestart">
       <property name="text">
        <string>Automatic restart after bus off condition</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
